tests/test_v2.py

```diff
@@ -0,0 +1,94 @@
+import os, sys, pytest
+import asyncio
+
+from plugins.pathfinder.app.pathfinder_svc import PathfinderService
+from plugins.pathfinder.app.parsers import caldera
+import networkx as nx
+
+TEST_REPORT_PATH = os.path.join(os.path.dirname(__file__), 'data/v2_algo_testcase.yml')
+PATHFINDER_V2_ID = 'deadbeef-6a18-4dcf-b659-79b8dd0d8d89'
+
+@pytest.fixture
+def test_report():
+    return caldera.ReportParser().parse(TEST_REPORT_PATH)
+
+
+@pytest.fixture
+def pathfinder_svc():
+    return PathfinderService(services={})
+
+@pytest.fixture
+def expected_graph(test_report):
+    graph = test_report.network_map
+    graph.remove_node('1-3')
+    return graph
+
+
+@pytest.mark.asyncio
+class TestV2:
+
+    async def test_generate_exploit_graph(self, test_report, pathfinder_svc, expected_graph):
+        '''
+            Tests the Pathfinder Service 'generate_exploitability_graph()'
+        '''
+        func_result = await pathfinder_svc.generate_exploitability_graph(test_report, [], [])
+        assert nx.is_isomorphic(expected_graph,func_result)
+
+    async def test_create_adversary_from_path(self, test_report, pathfinder_svc, mocker):
+        '''
+            Tests the Pathfinder Service 'create_adversary_from_path()'
+        '''
+        start = '1-1'
+        end = '2-3'
+        vuln_graph = await pathfinder_svc.generate_exploitability_graph(test_report, [], [])
```

Comment:
Getting an error saying PathfinderService doesn't have a method called `generate_exploitability_graph`, I can't find the method in the PathfinderService class myself either.

Reply:
The tests are branched off v2-main, which doesn't have v2-algo in yet, so they don't technically work in this branch. Chris and I wanted to split work up so v2-algo wouldn't get too big.